Schema/V1/no.ks.fiks.arkiv.v1.arkivering.arkivmelding.oppdater.xsd

```diff
@@ -24,12 +24,15 @@
     <xs:complexType name="mappeOppdatering">
         <xs:sequence>
             <xs:element name="referanseTilMappe" type="n5mdk:referanseTilMappe"/>
+            <xs:element name="referanseEksternNoekkel" type="n5mdk:eksternNoekkel" minOccurs="0"/>
```

Comment:
Ligger ikke denne i referanseTilMappe?

Reply:
Jo, men hvis man ønsker å endre `referanseEksternNoekkel` så bør vi skille mellom den som identifiserer den man ønsker å endre i arkivet og det man ønsker å endre den til? Altså i `referanseTilMappe` ligger den gamle `referanseEksternNoekkel` og i denne på linje 27 angir man den nye. Vi kunne forsøkt å tydeliggjøre det ved å kalle den `referanseEksterNoekkelOppdatering` eller noe sånt?

Reply:
Blir et spesialtilfelle der GI hadde egen tjeneste. Litt usikker på hva som er mest fornuftig, men det må uansett være tydelig skille på ny nøkkel og identifiserende nøkkel. Og i det tilfellet kallende system ikke har planer om å gjøre en oppdatering, blokkering av utilsiktet sletting av nøkkelen. Jeg heller mot egen tjeneste for nøkkeloppdatering selv om det gir ekstra tjenester. Det er sjelden dette trengs, tipper at tjenesten mest brukes for senere påføring av nøkkelen. Ta dette på torsdag?

Reply:
Det kan vi gjøre. Vi tar det opp i plenum på torsdag 👍